### ISSUE TITLE
Possible bug? : Using views/server groups

### ISSUE THREAD

**Issue by Lormen** [WillyXJ/facileManager#255]
I've been trying to get my setup running using a master and slave for both internal and external uses without much success.

All looks to be well but I can't see how to remove certain views from server configs, am I just missing something?

Views have been set up as Internal, External and Dynamic
Groups are just Internal and External (The Dynamic views uses the Internal group)
Templates for all three zones have been set to use these values 

I have tested removing the extra, unneeded views from named.conf and restarting named manually and then all is fine. Is there something I've missed?

**Comment by Lormen** [comment]
Don't worry, think I've sorted it. Loads of old config seemed to be getting in the way!

**Comment by Lormen** [comment]
Nope, that seemed to be a fluke, even with no other zones being about except the views with the include lines that contain no zones it fails. When other views are added the server ignores all.

**Comment by Lormen** [comment]
found what I needed in the end, missed the "All servers" section within the views.

**Comment by WillyXJ** [comment]
Oh good! I'm glad you figured it out as I was just getting a new test environment going to see if this scenario could be reproduced.  Yes, you can create views for all servers, server groups, or individual servers and that is true for most of the configuration.  Options can even be defined globally and then overridden per server group or server.

**Comment by Lormen** [comment]
I did still find it weird that if a view has an include for a zone file that is then empty (or not) it wouldn't continue down and properly load the rest of them. 

I even tested moving the External view to be first, leaving the rest of them in underneath and then it worked as expected... Seems more like an issue with my version of BIND than FM but it's a pretty weird bug...

I will open a fresh ticket for the other issue I've found which is FM related, only small though.

Thanks again!
